support-frontend/assets/components/subscriptionCheckouts/subscriptionSubmitButton.scss

```diff
@@ -0,0 +1,10 @@
+@import '~stylesheets/gu-sass/gu-sass';
+
+.component-submit-button {
+    margin-bottom: $gu-v-spacing;
+}
+
+.component-submit-button--margin {
+    max-width: gu-span(5);
+    margin: $gu-v-spacing/2 $gu-h-spacing/2 $gu-v-spacing * 2;
```

Comment:
Would be nice to be consistent with the spacing on `/2` and ` * 2` here and in the other scss file.

Reply:
We have the silly (at least I think so) pattern about a smaller top margin (6px) than bottom margin (12-24px) on UI. I'll check in on Lucy with this part @jfsoul

Reply:
I actually took this spacing from an existing component (FormSection), which I had to remove in order to make the error summary sit flush with its container. However, on standardising it, the spacing looks fine to me:
![Screen Shot 2019-08-21 at 10 42 54](https://user-images.githubusercontent.com/16781258/63421565-b22f5000-c400-11e9-8cee-2cae19f8ed41.png)

Reply:
Maybe a little spacey at the top. @jesseyuen - what do you think?

Reply:
It sounds like this is a useful conversation, but to be honest I was just talking about whitespace in the code itself 😄

Reply:
Heh, that's funny